test/functional/cursor.test.js

```diff
@@ -3752,7 +3752,7 @@ describe('Cursor', function () {
           db.s.topology,
           db.s.namespace,
           {},
-          { limit: 0, skip: 0, slaveOk: false, readPreference: 42 }
+          { limit: 0, skip: 0, secondaryOk: false, readPreference: 42 }
```

Comment:
This test is being skipped. I unskipped it, and it failed in the same way for both `slaveOk: false` and `secondaryOk: false`. I'm not sure how else to test this broken test.

Reply:
the comment above says that this test is questionable in value; and secondaryOk is not a valid command option for any command (slaveOk wasn't part of the public command options object); I think we can just delete the test